plugin/core/signature_help.py

```diff
@@ -115,7 +116,10 @@ def _render_label(self, view: sublime.View, signature: SignatureInformation) ->
                     # route relies on the client being smart enough to figure where the parameter is inside of
                     # the signature label. The above case where the label is a tuple of (start, end) positions is much
                     # more robust.
-                    start = label[prev:].find(rawlabel)
+                    label_match = re.search(r"\W{}\W".format(re.escape(rawlabel)), label[prev:])
+                    start = -1
+                    if label_match:
+                        start = label_match.start() + 1  # skip the first matched \W by adding + 1
```

Comment:
`main`:

<img width="743" alt="Screenshot 2022-09-28 at 12 25 47" src="https://user-images.githubusercontent.com/153197/192756141-cd40b62c-db9f-4ac4-b71b-585ace57d5da.png">

this branch:

<img width="738" alt="Screenshot 2022-09-28 at 12 26 07" src="https://user-images.githubusercontent.com/153197/192756147-0e5b047d-ff28-457c-b2b3-b564e2563473.png">

Reply:
To avoid having issues like this,

Prefer searching by word boundary, 
if a match is not found,
fallback to the previous logic for finding the parameter.

Reply:
Another alternative that I think should always work would be to use a negative lookbehind and negative lookahead for word characters in the regex: `r"(?<!\w){}(?!\w)"`

But the idea to fallback to the previous logic should be fine too.

Reply:
IMO not having a fallback logic is better.
So I went with your suggestion by using the regex.